projects/canopy/src/lib/variant/variant.notes.ts

```diff
@@ -0,0 +1,61 @@
+export const notes = `
+
+# Variant Directive
+
+## Purpose
+This directive allows you to add one of the five common colour variants to any Canopy component, such as \`\`info\`\` and \`\`success\`\`.
+
+They are already applied to existing Canopy components which use these variants out of the box, such [Alert](/?path=/story/components-alert--standard), [Detail](?path=/story/components-details--standard) and [Form Validation](?path=/story/components-form-validation--standard), but this directive allows you to use them anywhere where required.
```

Comment:
Should it be `such as Alert`? just wondering!

Reply:
Fixed 👍